spec/presenters/two_factor_auth_code/piv_cac_authentication_presenter_spec.rb

```diff
@@ -25,7 +25,46 @@ def presenter_with(arguments = {}, view = ActionController::Base.new.view_contex
         app: content_tag(:strong, APP_NAME))
     end
 
-    it { expect(presenter.help_text).to eq expected_help_text }
+    context 'with AAL3 required, and only one method enabled' do
+      let(:expected_help_text) do
+        t('instructions.mfa.piv_cac.confirm_piv_cac_only_html')
+      end
+      it 'finds the PIV/CAC only help text' do
+        aal3_policy = instance_double('AAL3Policy')
+        allow(aal3_policy).to receive(:aal3_required?).and_return true
+        allow(aal3_policy).to receive(:multiple_aal3_configurations?).and_return false
+        allow(presenter).to receive(:aal3_policy).and_return aal3_policy
```

Comment:
maybe move this `allow` into a `before` for the whole test? and then make `aal3_policy` a `let` so you can override it where you need to? Or override the specifc methods? That way it can always be around and we don't have to stub it in every test case that touches it

```ruby
  let(:aal3_policy) do
    instance_double('AAL3Policy',
      aal3_required?: aal_required,
      multiple_aal3_configurations?: multiple_configurations
    )
  end
  before { allow(presenter).to receive(:aal3_policy).and_return(aal3_policy)

  context 'when aal3 is required but there are not multiple configurations' do
    let(:aal3_required) { true }
    let(:multiple_configurations) { false }

    it 'does the thing' do
      # ...
    end
  end
```